app/javascript/src/components/Dashboard/Notes/NotesMenu.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import { Search, Settings, Plus } from "@bigbinary/neeto-icons";
+import { Typography } from "neetoui/v2";
+import { MenuBar } from "neetoui/v2/layouts";
+
+const NotesMenu = ({ showNotesMenu }) => {
+  return (
+    <MenuBar showMenu={showNotesMenu} title="Notes" className="min-h-full">
+      <MenuBar.Block label="All" count={200} active />
+      <MenuBar.Block label="Users" count={80} />
+      <MenuBar.Block label="Leads" count={60} />
+      <MenuBar.Block label="Visitors" count={60} />
+      <MenuBar.SubTitle iconProps={[{ icon: Search }]}>
+        <Typography
+          component="h4"
+          style="h5"
+          textTransform="uppercase"
+          weight="bold"
+        >
+          Segments
+        </Typography>
+      </MenuBar.SubTitle>
+      <MenuBar.Block label="Europe" count={80} />
+      <MenuBar.Block label="Middle-East" count={60} />
+      <MenuBar.Block label="Asia" count={60} />
+      <MenuBar.SubTitle
+        iconProps={[
+          {
+            icon: Settings,
+          },
+          {
+            icon: Plus,
+          },
+          {
+            icon: Search,
+          },
+        ]}
+      >
+        <Typography
+          component="h4"
+          style="h5"
+          textTransform="uppercase"
+          weight="bold"
+        >
+          Tags
+        </Typography>
+      </MenuBar.SubTitle>
+
+      <MenuBar.Block label="Sales" count={80} />
+      <MenuBar.Block label="Finance" count={60} />
+      <MenuBar.Block label="User Experience" count={60} />
+    </MenuBar>
+  );
+};
+
+export default NotesMenu;
```

Comment:
Since the file is inside `Notes`, we don't need to specifically prefix or suffix file/component names with `Notes`. 
We can simply name this as `MenuBar.jsx` or `Menu.jsx`